lib/rdf.js

```diff
@@ -127,6 +127,28 @@ function createRDF(manifest) {
     });
   }
 
+  if (manifest.locales) {
+    for (var locale in manifest.locales){
```

Comment:
please add a space between `)` and `{`, your tests are actually failing because of that